lld/MachO/ObjC.cpp

```diff
@@ -349,11 +349,15 @@ void objc::checkCategories() {
 namespace {
 
 class ObjcCategoryMerger {
+  // In which language was a particular construct originally defined
+  enum SourceLanguage { ObjC, Swift, Unknown };
```

Comment:
It seems natural to start with an invalid case (0) for `Unknown`.
```suggestion
  enum SourceLanguage { Unknown, ObjC, Swift };
```